docs/src/main/sphinx/connector/clickhouse.rst

```diff
@@ -173,9 +173,10 @@ Type mapping
 ------------
 
 Because Trino and ClickHouse each support types that the other does not, this
-connector modifies some types when reading or writing data. Data types may not
-map the same way in both directions between Trino and the data source. Refer to
-the following sections for type mapping in each direction.
+connector :ref:`modifies some types <type-mapping-overview>` when reading or
+writing data. Data types may not map the same way in both directions between
```

Comment:
Data types may not map the same way between Trino and the data source.

Reply:
That sounds wrong @Jessie212 

But I think we should call this out even more specifically... not sure the "direction" model is really correct..

Reply:
The direction is correct. Note that saying `Data types may not map the same way in both read and write queries` is not correct since even in read queries the "write" mappings get used for things like predicate pushdown.

Reply:
This sounds to me like the sentence is saying both between the two directions and also in both directions, which sound redundant to me. And if we're talking about mapping from Trino to the data source and vice versa, isn't it true that mapping may not be the same?

Reply:
But its not about directions at all .. I would really like to find a better way  .. 

one mapping is about interactions with existing columsn (read and write)

the other mapping is about creating columns (ddl)

Reply:
This paragraph is currently boiler-plate for most type mapping sections, so if we want to make a change to one we should then apply it to all those sections. I think that's probably a discussion for another commit, out of scope for this overview PR.

Reply:
"read" and "write" is the simplest explanation. i.e. any operation which requires converting from a Trino value to data source value uses the "write" mapping (e.g. WHERE conditions with literals, CTAS, INSERT) while any operation requiring converting from data source value to Trino value uses the "read" mapping (SELECT is the only one that matters).

Agree with Joe to focus on this in different PR - I'm not sure what's the best way to handle/explain this right now.